app/src/main/java/org/fossasia/openevent/app/ui/views/DatePicker.java

```diff
@@ -5,29 +5,35 @@
 import android.text.TextUtils;
 import android.util.AttributeSet;
 
+import org.fossasia.openevent.app.data.event.serializer.ObservableString;
 import org.fossasia.openevent.app.utils.DateUtils;
 import org.threeten.bp.LocalDate;
 import org.threeten.bp.LocalDateTime;
 
 public class DatePicker extends AbstractDateTimePicker {
+    private final ObservableString value;
```

Comment:
This isn't what I said, first solution was perfect. I wanted the field to be extracted locally in the function so that getValue is not called repeatedly